README.md

```diff
@@ -26,6 +26,13 @@ git clone https://github.com/vlang/sdl.git ~/.vmodules
 cd ~/.vmodules/sdl
 git checkout 2.0.12
 ```
+and for Windows:
+```bash
+git clone https://github.com/vlang/sdl.git %HOMEPATH%/.vmodules/sdl
```

Comment:
This is the same as `v install sdl`, which should probably be used since it's "the V way".